src/phoenix/server/main.py

```diff
@@ -119,6 +120,7 @@ def _get_pid_file() -> Path:
     parser.add_argument("--export_path")
     parser.add_argument("--host", type=str, required=False)
     parser.add_argument("--port", type=int, required=False)
+    parser.add_argument("--host-root-path", type=str, required=False)
```

Comment:
Not sure we need the command line argument here in addition to the environment variable. We've been deprecating command line arguments such as `--enable-prometheus` that have environment variable equivalents if setting them via environment variables feels more natural, e.g., in Kubernetes. I think host root path probably falls into that category.

Reply:
Agreed, I removed the CLI parameter.